web/css/timeline.css

```diff
@@ -708,3 +735,148 @@ button:focus {
 .datepicker.android-dark .datepicker-scroll li.disabled {
   color: rgb(88, 88, 88) !important;
 }
+
+.datepicker-navbar-btn:hover {
+  background: #fff;
+}
+
+/* TIMELINE DATA PANEL */
+
+.timeline-data-panel-container {
+  position: absolute;
+  background: rgba(40, 40, 40, 1);
+  border-top-right-radius: 5px;
+  border-top-left-radius: 5px;
+}
+
+.animate-timeline-data-panel-slide-up {
+  z-index: 5;
+  bottom: 66px;
+  opacity: 1;
+}
+
+.animate-timeline-data-panel-slide-down {
+  z-index: -5;
+  bottom: -12px;
+  opacity: 0;
+}
+
+.timeline-data-panel i.wv-close:hover {
+  color: #999;
+  cursor: pointer;
+}
+
+.timeline-data-panel-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: thin solid #808080;
+  padding: 5px;
+}
+
+.timeline-data-panel-header-title {
+  width: 150px;
+  padding: 5px;
+}
+
+.label-layer-active {
+  color: #000;
+}
+
+.label-layer-inactive {
+  color: #999;
+}
+
+#wv-toggle-data-matching-main {
+  align-items: center;
+  margin: 0;
+  white-space: nowrap;
+  flex-grow: 1;
+}
+
+#wv-toggle-data-matching-main .react-switch-case {
```

Comment:
Best to avoid adding (and never hurts to replace) `#` id selectors when possible, even though they are all over the codebase.

I would also suggest adding a `.thin-border` class  in the switch component and putting the styles in switch.js component so they are reusable, this would make it it so you can just pass a border prop in the react component.
```
.switch-thin-border .react-switch-label {
  border: thin solid rgb(97, 97, 97);
  height: 17px; // account for added pixel height
  width: 32px; // account for added pixel width
}
```

Reply:
I modified the switch and id styling.

For the striped bar, I'm not sure how to go about addressing this to get 100% striped bar movement on large coverage bars.

The issue is that large bars are not fully rendered. So for your above issue, it would be a massive bar rendered starting in 2000 to current time if they were fully rendered. For a fix to handle the partially rendered larger bars, an overlay is added to larger bars to simulate the revised striped bars being dragged. What you are seeing where it doesn't drag is that on the edges of large bar coverage the overlay is removed so the simulated striped bars are not being dragged any longer.

The code is https://github.com/nasa-gibs/worldview/blob/layer-data-availability/web/js/components/timeline/timeline-data/layer-data-items.js#L198-L208
Any ideas on how to address this without fully rendering the entire bars?